Accessibility updates to codelab.md, accessibility.md (#7678)
flutter/website: src/get-started/codelab.md

  * A physical device ([Android][] or [iOS][]) connected to your
    computer and set to developer mode
  * Screen reader enabled on the physical device (<a href="#talkback">TalkBack</a> on Android, <a href="#talkback">VoiceOver</a> on iPhone)
  * The [iOS simulator][] (requires installing Xcode tools)
  * The [Android emulator][] (requires setup in Android Studio)

>   * A browser (Chrome is required for debugging)

Reviewer: **suggestion:** Active voice.
```suggestion
  * A browser (Debugging requires the Chrome browser.)
```

  a `Text` child widget. The Center widget aligns its widget subtree
  to the center of the screen.

{{site.alert.note}}

>       The app automatically creates a semantics tree for screen readers. 
>       Each node in the tree can correspond to one or several 
>       widgets and can be <a href="https://api.flutter.dev/flutter/widgets/Semantics-class.html">further customized</a> to tell screen readers how to behave with the node. 

Reviewer: **suggestion:** Make the copy more concise and use semantic line breaks.
```suggestion
      The app creates a semantics tree for screen readers. 
      Each node in the tree can correspond to one or several 
      widgets. These nodes can be <a href="https://api.flutter.dev/flutter/widgets/Semantics-class.html">customized</a>
      to tell screen readers how to behave with the node. 
```

      "UpperCamelCase".
    {{site.alert.end}}

    {{site.alert.important}}
    Using pascal case also helps screen readers identify the individual words 
    in the compound word and provides a better 
    experience to visually impaired users.

>     {{site.alert.end}}

Reviewer: **issue:** Refer to this capitalization format as **Upper Camel Case** per the [Dart style guide](https://dart.dev/guides/language/effective-dart/style#do-name-types-using-uppercamelcase). Remove the mention of "pascal case".

      {% include docs/ios-windows-figure-pair.md image="step2.png" alt="App at completion of second step" %}
    {% endindent %}

>     **You can also try out the app with a screen reader. All you need to do is turn on the screen reader on your device by following the steps below:** 

Reviewer: **issues:** 
- Replace "below" as it runs counter to [Google's documentation accessibility guidelines](https://developers.google.com/style/accessibility).
- Use [semantic line breaks](https://sembr.org/) for copy.

```suggestion
    **You can try the app with a screen reader.
    Turn on the screen reader on your device using the following steps:** 
```

    {% comment %} Nav tabs {% endcomment -%}
    <ul class="nav nav-tabs" id="editor-setup" role="tablist">
      <li class="nav-item">
        <a class="nav-link active" id="talkback-tab" href="#talkback" role="tab" aria-controls="talkback" aria-selected="true">TalkBack on Android</a>
      </li>
      <li class="nav-item">
        <a class="nav-link" id="voiceover-tab" href="#voiceover" role="tab" aria-controls="voiceover" aria-selected="false">VoiceOver on iPhone</a>
      </li>
    </ul>

    {% comment %} Tab panes {% endcomment -%}
    <div class="tab-content">
    
    <div class="tab-pane active" id="talkback" role="tabpanel" aria-labelledby="talkback-tab" markdown="1">

    <!-- **Turn on TalkBack on your Android phone to try the app with a screen reader** -->

    1. On your device, open Settings. 
    2. Select Accessibility and then TalkBack.
    3. Turn 'Use TalkBack' on or off.
    4. Select Ok.

>     You can also view this video to learn how to find and customize all accessibility features provided by Android.

Reviewer: **issue:** [Avoid passive voice](https://developers.google.com/style/voice).

```suggestion
    To learn how to find and customize all accessibility features that Android provides, view this video.
```

  containing the saved favorites.
* Modify the theme color, making an all-white app.

## Explore the Flutter SDK

> * <a href="https://docs.flutter.dev/get-started/flutter-for/react-native-devs">Flutter for React Native developers</a>

Reviewer 2: Was this change intended? Seems unrelated to the rest of the change.

Reviewer 3: Build is green! We should be good to land once this comment is resolved.

Reviewer 2: I'd revert this part of the change as it's an unrelated change. If it's required, please fold into a new PR. Alternatively, it might be guff from Git history shenanigans...
